miio/viomivacuum.py

```diff
@@ -140,9 +156,8 @@ def has_new_map(self) -> bool:
         return bool(self.data["has_newmap"])
 
     @property
-    def is_mop(self) -> bool:
-        """True if mopping."""
-        return bool(self.data["is_mop"])
+    def is_mop(self) -> ViomiMopMode:
+        return ViomiMopMode(self.data["is_mop"])
```

Comment:
Should this be called `mop_mode` instead of `is_mop`? Also, please add a docstring for the property, too.

Reply:
Done.